### PR TITLE
fix material

### PR DESCRIPTION
**Pull request by modos189**

### CODE REVIEW

**Comment by nicosomb** [comment]
yeah, good job guy. I like the notifications :+1:

File: src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig [hunk 1]
```diff
@@ -18,6 +18,67 @@
...
+                </ul>
+            </div>
+            <form method="get" action="index.php">
+                <div class="input-field nav-panel-search" style="display: none">
+                    <input name="search" id="searchfield" type="search" required placeholder="{% trans %}Enter your search here{% endtrans %}">
+                    <label for="search"><i class="mdi-action-search"></i></label>
+                    <i class="mdi-navigation-close"></i>
+                </div>
+            </form>
+            <!--<form name="entry" method="post" action="{{ path('new_entry') }}">
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig]: "add a new link" form will be displayed in the header? Or is that dead code?

File: src/Wallabag/CoreBundle/Resources/views/themes/material/public/js/init.js [hunk 2]
```diff
@@ -10,19 +10,26 @@ $(document).ready(function(){
     //    $(".nav-panel-buttom").hide(100);
     //    $(".nav-panel-add").show(100);
     //    $(".nav-panel-menu").hide(100);
+    //    $(".nav-panels .action").hide(100);
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/public/js/init.js]: dead code?

**Comment by nicosomb** [comment]
just 2 comments, we can merge after that.

**Comment by modos189** [comment]
When you are able to add material I uncomment.

If I understand correctly, it is now possible to add articles from the page ONLY / Neuve, because you want to pass the token

**Comment by nicosomb** [comment]
> If I understand correctly, it is now possible to add articles from the page ONLY http://site.com/new,

yes, you can add new articles from this page only. So you can implement form in the header I think.

**Comment by modos189** [comment]
Please tell me how to do it

**Comment by nicosomb** [comment]
OK, I'll do that after my current work.

**Comment by j0k3r** [comment]
Also, could you rebase your branch against the v2 to fix conflicts?

**Comment by modos189** [comment]
Up

**Comment by nicosomb** [comment]
I merged your PR. About new link form, my PR #1307 is not yet finished, so I think it's better for you to work on a new PR.

**Pull request merged**